api/swagger.yaml

```diff
@@ -332,6 +332,7 @@ definitions:
       Memory:
         description: "Memory limit in bytes."
         type: "integer"
+        format: "int64"
```

Comment:
Looks like `CpuShares` is an int64 as well;

https://github.com/moby/moby/blob/f28cb422e69bd4239401e6ea32b65b56485d6691/api/types/container/host_config.go#L308-L312